docs/core/tutorials/aspnet-core.md

```diff
@@ -7,4 +7,4 @@ ms.date: 06/20/2016
 ---
 # Getting started with ASP.NET Core
 
-For tutorials about developing ASP.NET Core web applications, we suggest you head over to [ASP.NET Core documentation](/aspnet/core/).
+For tutorials about developing ASP.NET Core web applications, we suggest you head over to [ASP.NET Core documentation](https://docs.microsoft.com/en-us/aspnet/core/tutorials).
```

Comment:
Links such as `/aspnet/core...` are resolved by the publishing platform for docs.microsoft.com. So the /aspnet/core/ part of the link is correct. It should link to tutorials, though. So a better link is /aspnet/core/tutorials/. Would you mind changing it?

Also, and totally unrelated to your PR, "head over to" is a difficult phrase for localization. Would you mind changing it to "see"?